lms/static/sass/inline-discussion-rtl.scss

```diff
@@ -0,0 +1,23 @@
+// From lms-main-v1-rtl.scss
```

Comment:
Thanks. I'd remove the comment now, and replace it with a header that says what this file is.